packages/map-popup/src/styled.ts

```diff
@@ -1,11 +1,11 @@
 import styled from "styled-components";
 
 /* eslint-disable-next-line import/prefer-default-export */
-export const ViewStopButton = styled.button<{ stopId?: string }>`
+export const ViewStopButton = styled.button<{ stopId?: string | boolean }>`
```

Comment:
Wait I'm confused, if this is trying to account for no `stopId`, then couldn't it just be `stopId?: string` and the border check below is just checking if `stopId` exists? This feels a little convoluted as is